### ISSUE TITLE
Provide population data with cumulative values

### ISSUE THREAD

**Issue by lisphilar** [globaldothealth/monkeypox#179]
Population data is required to estimate the impact of the number of cases at the selected location, and to analyze the number of cases with SIR-derived math models. This is related to #178.

So, it will be great to provide population data of each country/city or document how to get population data.

**Comment by jim-sheldon** [comment]
We can certainly include retrieving population data as part of automated script(s). There are external APIs and databases we can pull from.

**Comment by jim-sheldon** [comment]
OWID also provides population data as a supplement to ours: https://ourworldindata.org/monkeypox

**Comment by lisphilar** [comment]
Yes. Population values used by OWID is saved [here](https://github.com/owid/covid-19-data/blob/master/scripts/input/un/population_latest.csv) and the primary source (CC BY 3.0 IGO) is as follows.
https://population.un.org/wpp/Download/Standard/CSV/

Or, we can get total values from Word Bank Data.
https://data.worldbank.org/indicator/SP.POP.TOTL

**Comment by lisphilar** [comment]
OWID and "The Word Bank Data" use values in 2021.
We can get estimted total value of each country in 2022 using WPP2022_TotalPopulationBySex.csv

![image](https://user-images.githubusercontent.com/7270139/188280921-3dab8c85-20e5-4237-8ab3-a020ec52f50b.png)

Note it will be difficult to get population values at city level.

**Comment by jim-sheldon** [comment]
The United Nations provides [city-level data](https://data.un.org/Data.aspx?d=POP&f=tableCode%3A240), though it is not immediately clear whether they use ISO-3166.

**Comment by oliszymanski** [comment]
> The United Nations provides [city-level population data](https://data.un.org/Data.aspx?d=POP&f=tableCode%3A240), though it is not immediately clear whether they use ISO-3166.

not all data is provided. When I filtered for Poland in 2022. Not sure how about for the other countries but quite possible that there are many of them. Maybe in the future they will gather more data but as for now, it isn't much but we can use the existing one ofc

**Comment by oliszymanski** [comment]
How about worldometers?
https://www.worldometers.info/world-population/

**Comment by lisphilar** [comment]
@jim-sheldon and @oliszymanski 
Thank you for your comments!

Worldometers says the primary source regarding country-level data is [World Popularation Prospects 2022 by UN](https://population.un.org/wpp/Download/Standard/CSV/).

City-level data is also available and can be scraped. I could not find useful APIs and URLs of primary sources.
https://www.worldometers.info/world-population/poland-population/

At this time, city-level confirmed cases data is not included in "Timeseries" and "Timeseries by country" of this repository.
As the first step of this issue, is it possible to add only country-level population values to "Timeseries by country" file?

**Comment by jim-sheldon** [comment]
I support this. Created https://github.com/globaldothealth/monkeypox/issues/217 to capture the work.
